SimTKcommon/Simulation/include/SimTKcommon/internal/StateImpl.h

```diff
@@ -35,6 +35,100 @@ contain any user-visible objects. **/
 
 namespace SimTK {
 
+//==============================================================================
+//                             DEPENDENT LIST
+//==============================================================================
+/* This class contains a set of downstream cache entries that are dependent 
+on the value of some prerequisite that is the owner of this object. 
+Prerequisites may be 
+  - continuous state variables q,u,z
+  - discrete variables, or
+  - upstream cache entries. 
+
+When the prerequisite's value changes or is explicitly invalidated, the
+notePrerequisiteChange() method is invoked, causing the invalidate() method of 
+each of the dependent cache entries to be called.
+
+CAUTION: upstream cache entries may be invalidated implicitly by their
+depends-on stage becoming invalid. For this to work, the depends-on stage
+of a downstream cache entry must be the same as the highest depends-on stage of 
+any of it upstream cache prerequisites. That way it is guaranteed to be 
```

Comment:
> of it -> of its

Thanks, Tom. Fixed in [this commit](https://github.com/simbody/simbody/commit/c12503798a9a353c083f1db4ca851da78d13fcad) along with another problem I noticed while fixing it.